package.json

```diff
@@ -33,7 +33,7 @@
     "build": "jlpm run build:dev",
     "build:all": "lerna run build:all",
     "build:builder": "cd builder && jlpm run build",
-    "build:core": "cd jupyterlab/staging && jlpm && (jlpm deduplicate || jlpm) && jlpm run build",
+    "build:core": "cd jupyterlab/staging && jlpm && (jlpm yarn-deduplicate || jlpm) && jlpm run build",
```

Comment:
Should we add the options of the `duplicate` script?

```
"deduplicate": "jlpm yarn-deduplicate -s fewer --fail",
```

Reply:
Sounds good :+1:

Reply:
```suggestion
    "build:core": "cd jupyterlab/staging && jlpm && (jlpm yarn-deduplicate -s fewer --fail || jlpm) && jlpm run build",
```